app/controllers/admin/settings/project_custom_fields_controller.rb

```diff
@@ -181,5 +218,9 @@ def drop_success_streams(call)
         update_section_via_turbo_stream(project_custom_field_section: call.result[:old_section])
       end
     end
+
+    def include_sub_projects?
+      ActiveRecord::Type::Boolean.new.cast(permitted_params.project_custom_field_project_mapping[:include_sub_projects])
```

Comment:
:yellow_circle: Should we avoid `permitted_params` in favor of simple `params`? It seems to be not needed here, because there is no mass assignment.